templates/terraform/unordered_list_customize_diff.erb

```diff
@@ -0,0 +1,37 @@
+keys := diff.GetChangedKeysPrefix(<%= go_literal(Google::StringUtils.underscore(prop.name)) -%>)
+if len(keys) > 0 {
```

Comment:
nit: I would personally find it easier to follow if it were
```
if len(keys) == 0 {
  return nil
}
```
but up to you

Reply:
Agreed, fixed and working its way through.